Samples/AppLifecycle/Instancing/cs-winui-packaged/CsWinUiDesktopInstancing/ActivationTrackerHelper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.Win32;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CsWinUiDesktopInstancing
+{
+    internal static class ActivationTrackerHelper
+    {
+        private const string c_redirectionKeyName = @"SOFTWARE\AppLifecycleExample";
+        private const string c_redirectionValueName = "NumberOfActivations";
+
+        private static RegistryKey GetRegistryKey()
+        {
+            return Registry.CurrentUser.CreateSubKey(c_redirectionKeyName, true);
+        }
+
+        private static void SetNumberOfActivations(int newNumberOfActivations)
+        {
+            RegistryKey instanceTrackerKey = GetRegistryKey();
+            instanceTrackerKey.SetValue(c_redirectionValueName, newNumberOfActivations);
+        }
+
+        internal static int IncrementAndSetNumberOfRedirectionsReturnNewNumber()
```

Comment:
the name is a little odd - "IncrementAndSet*" and "*ReturnNewNumber". I'm not familiar with this level of detail in our api guidance. Given its internal/private, maybe not too bad?

Reply:
Yeah.  It is a bit wordy.  I just wanted to be descriptive on what the method does including all side-effects.

But, as you said, because this is internal and not production ready code, I think the smaller name works here.